Chap_API_Data_Mgmt.tex

```diff
@@ -261,23 +264,22 @@ \subsection{\code{PMIx_Data_unpack}}
 %%%%
 \descr
 
-The unpack function unpacks the next value (or values) of a specified type from the given buffer. The buffer must have already been initialized via an \refmacro{PMIX_DATA_BUFFER_CREATE} or \refmacro{PMIX_DATA_BUFFER_CONSTRUCT} call (and assumedly filled with some data) --- otherwise, the unpack_value function will return an error. Providing an unsupported type flag will likewise be reported as an error, as will specifying a data type that \textit{does not} match the type of the next item in the buffer. An attempt to read beyond the end of the stored data held in the buffer will also return an error.
-
```

Comment:
Make a decision on whether we want to describe behavior of debug vs non-debug and whether to mention that type checking is only done for debug.   Either way, don't may claim that type matching is done in all cases.

Reply:
We have gone in the direction of not describing debug behavior